.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```

Comment:
An empty template is probably obsolete since one can select "Open a blank issue" to have an empty issue. That little link however might be overseen by some users, compared to a large button besides the other templates.